app/assets/javascripts/index.js

```diff
@@ -187,8 +187,10 @@ $(document).ready(function () {
     $.cookie('_osm_location', OSM.locationCookie(map), { expires: expiry, path: '/' });
   });
 
-  if ($.cookie('_osm_welcome') === 'hide') {
-    $('.welcome').hide();
+  if ($.cookie('_osm_welcome') !== 'hide' &&
+     (typeof window.matchMedia !== "function" ||
+     !window.matchMedia("only screen and (max-width:768px)").matches)) {
```

Comment:
This doesn't appear to have anything to do with resolving #1951 so perhaps you could explain what it's about?

How widely supported is that function? I mean I can see you're concerned it might not exist so it's obviously not universal - checking it's type is probably overkill in any case unless we're expecting somebody to create `window.matchmedia` with some other type to annoy us!

Reply:
Sure. This may really be kind of an overkill, and it's only useful for mobile devices anyway. I'm perfectly fine to remove those two lines 191+192 again. Their only purpose is to skip `$('.welcome').show()`, if the screen is too small. 

Reason being that small.scss defines the following `display:none` with `!important`. So whatever we try to do in terms of show(), it gets overruled immediately (with some flickering on the mobile screen again).

```
  #sidebar .welcome,
  #sidebar #banner {
    display: none !important;
  }
```

According to https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia#Browser_compatibility, window.matchMedia support is quite good. Line 191 should cover those browsers, where it's not yet supported.